clusterdock/cli.py

```diff
@@ -56,6 +56,21 @@ def main():
     build_parser = action_subparsers.add_parser('build',
                                                 formatter_class=FORMATTER_CLASS,
                                                 add_help=False)
+    build_parser.add_argument('--namespace',
+                              help='Namespace to use when looking for images',
+                              metavar='ns')
+    build_parser.add_argument('--network',
+                              help='Docker network to use',
+                              default=defaults['DEFAULT_NETWORK'],
+                              metavar='nw')
+    build_parser.add_argument('-o', '--operating-system',
+                              help='Operating system to use for cluster nodes',
+                              metavar='sys')
+    build_parser.add_argument('-r', '--registry',
```

Comment:
Same comment as above. To be honest, we may want to get rid of separate `--registry` and `--namespace` arguments and replace them to with `--repository` so people (or topologies) would just specify a repository once and for all and simplify a lot of stuff. What do you think? We'd need to also update topologies but I think it'd be worth it.

Reply:
as discussed offline, removed 'registry' and 'namespace' args and now have 'repository' arg